translations/de-ch.yml

```diff
@@ -0,0 +1,128 @@
+---
+de-ch:
+  title: "Bitcoin Cash - Die elektronischi Peer-to-Peer Währig"
+  meta:
+    description: |
+      Bitcoin Cash bütet dä gsamte Wält e solidi Währig. Dä Nutzä für Händler oder Chundä sind zueverlässigi Transaktione mit gringe Transaktionsköschtä. D Zuekunft für Bitcoin Cash glänzt dank unltimitiertem Wachstum, globalä Akzeptanz, Innovation und dezentralä Entwicklig.
+    twitter:
+      title: "Bitcoin Cash - Die elektronischi Peer-to-Peer Währig"
+      description: |
+        Bitcoin Cash bütet dä gsamte Wält e solidi Währig. Dä Nutzä für Händler oder Chundä sind zueverlässigi Transaktione mit gringe Transaktionsköschtä. D Zuekunft für Bitcoin Cash glänzt dank unltimitiertem Wachstum, globalä Akzeptanz, Innovation und dezentralä Entwicklig.
+    og:
+      title: "Bitcoin Cash - Die elektronischi Peer-to-Peer Währig"
+      description: |
+		Bitcoin Cash bütet dä gsamte Wält e solidi Währig. Dä Nutzä für Händler oder Chundä sind zueverlässigi Transaktione mit gringe Transaktionsköschtä. D Zuekunft für Bitcoin Cash glänzt dank unltimitiertem Wachstum, globalä Akzeptanz, Innovation und dezentralä Entwicklig.
```

Comment:
Spacing here is wrong. Need to make sure it is indented exactly 2 spaces like the other descriptions.